flags/src/main/java/com/yahoo/vespa/flags/PermanentFlags.java

```diff
@@ -296,6 +296,13 @@ public class PermanentFlags {
             "Takes effect at redeployment",
             ZONE_ID, APPLICATION_ID);
 
+    public static final UnboundListFlag<String> DISABLED_DEPLOYMENT_ZONES = defineListFlag(
```

Comment:
Why not boolean flag with `ZONE_ID` dimension?

Reply:
AFAIK that zone dimension is the zone of the requestor, but that's always the controller zone in this case?

Reply:
Ah, maybe that's not the case ... Then that's obviously the right thing.